windows/msvc/paths.props

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PyPathsIncluded>True</PyPathsIncluded>
+
+    <!-- The properties below specify the output directory structure.
+         This defaults to, for example for Configuration = Debug and Platform = x64:
+
+         micropython [PyBaseDir]
+             ├── ...
```

Comment:
Good question but I'm not aware of any problems. msbuild is quite picky about what you put in the files and it loads this one so it should be ok. I actually just copy-pasted the output from the tree command but if you want I can replace it with pure ascii characters